src/documents/InfusionEventSystem.md

```diff
@@ -262,14 +263,14 @@ var myFirer = fluid.makeEventFirer(options);
     </tr>
   </thead>
   <tr>
-    <td><code>preventable</code> (optional) DEPRECATED</td>
+    <td><code>preventable</code> [optional] DEPRECATED</td>
```

Comment:
Line 251 says: **The signature of `fluid.event.makeEventFirer` is not stable and will be revised in the 2.0 release of Infusion.** If the revised part is the deprecated `preventable` argument, shall this line be removed?

Also, the structure of `options` argument supplied to `fluid.makeEventFirer()` function is not clear. The doc probably wants to indicate that it's an object including members of elements listed in this table.